src/cli/helper.py

```diff
@@ -9,3 +10,9 @@ def exist_or_create(folder):
         except OSError as exc:
             if exc.errno != errno.EEXIST:
                 raise
+
+
+def age_from_timestamp(timestamp):
+    return str(
```

Comment:
Would be nice if we could properly format a `timedelta` ([kind of an unsolved problem][0]), as I find this actual `str` representation of timedeltas in Python a bit confusing (my intuition says about the time bit: looks like a date/time, not a span/period). But definitely better than not having an age at all. Of course, I have no clue what the ideal format should be :) maybe something `A days, B hours, C minutes, D seconds` (possibly omitting zeroes at the front or hours and seconds when we have reached days)

[0]: https://stackoverflow.com/questions/538666/format-timedelta-to-string

Reply:
Maybe we should go with the kubectl format? I kinda like it:
```
9m59s
10m
1h
3d
```

Reply:
@SteinRobert 
Yeah, that's pretty much what I had in mind. That will be clearer!

Reply:
@schwobaseggl sorry it took so long. Updated :)